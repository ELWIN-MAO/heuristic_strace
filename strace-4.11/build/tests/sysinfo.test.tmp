
mym_waitpid=102997

mym_waitpid=102997

mym_waitpid=102997

syscall_enter_mym_scno=59  sys_name=execve

mym_waitpid=102997

mym_waitpid=102997

syscall_exit_mym_scno=59  sys_name=execve

mym_waitpid=102997

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=102997

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=102997

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=102997

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=102997

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=102997

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=102997

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=102997

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=102997

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=102997

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=102997

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=102997

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=102997

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=102997

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=102997

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=102997

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=102997

syscall_enter_mym_scno=0  sys_name=read

mym_waitpid=102997

syscall_exit_mym_scno=0  sys_name=read

mym_waitpid=102997

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=102997

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=102997

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102997

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102997

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=102997

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=102997

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102997

syscall_enter_mym_scno=158  sys_name=arch_prctl

mym_waitpid=102997

syscall_exit_mym_scno=158  sys_name=arch_prctl

mym_waitpid=102997

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102997

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102997

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102997

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102997

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102997

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102997

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=102997

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=102997

syscall_enter_mym_scno=99  sys_name=sysinfo
sysinfo(
mym_waitpid=102997

syscall_exit_mym_scno=99  sys_name=sysinfo
{uptime=149875, loads=[70048, 27520, 17248], totalram=2079223808, freeram=285249536, sharedram=42409984, bufferram=181714944, totalswap=2144333824, freeswap=2141097984, procs=337, totalhigh=0, freehigh=0, mem_unit=1}
mym_scall_rval=0
) = 0

mym_waitpid=102997

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=102997

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=102997

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=102997

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=102997

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=102997

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=102997

syscall_enter_mym_scno=1  sys_name=write

mym_waitpid=102997

syscall_exit_mym_scno=1  sys_name=write

mym_waitpid=102997

syscall_enter_mym_scno=231  sys_name=exit_group

mym_waitpid=102997
+++ exited with 0 +++
