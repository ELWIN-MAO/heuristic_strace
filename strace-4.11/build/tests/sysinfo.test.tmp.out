sysinfo({uptime=149875, loads=[70048, 27520, 17248], totalram=2079223808, freeram=285249536, sharedram=42409984, bufferram=181714944, totalswap=2144333824, freeswap=2141097984, procs=337, totalhigh=0, freehigh=0, mem_unit=1}) = 0
+++ exited with 0 +++
lo IP Address 127.0.0.1
ens33 IP Address 192.168.159.137
IP Address 127.0.0.1
IP Address 192.168.159.137
